forms/Form.php

```diff
@@ -152,17 +184,25 @@ class Form extends RequestHandler {
 	 */
 	protected $attributes = array();
 
+	/**
+	 * @var array
+	 */
 	private static $allowed_actions = array(
-		'handleField', 
+		'handleField',
 		'httpSubmission',
 		'forTemplate',
 	);
 
 	/**
+	 * @var bool
+	 */
+	protected $securityTokenAdded;
```

Comment:
This is a new property, set in https://github.com/silverstripe/silverstripe-framework/pull/4274/files#diff-e7adb20524333364f626b479ceeded9cR663 It really looks like it should be on this object, not a parent object.